core/capabilities/remote/executable/server.go

```diff
@@ -196,8 +198,10 @@ func (r *server) getMessageHash(msg *types.MessageBody) ([32]byte, error) {
 	}
 
 	for _, path := range r.config.RequestHashExcludedAttributes {
-		if !req.Inputs.DeleteAtPath(path) {
-			return [32]byte{}, fmt.Errorf("failed to delete attribute from map at path: %s", path)
+		if req.Inputs != nil {
+			if !req.Inputs.DeleteAtPath(path) {
```

Comment:
@ettec Can we also be permissive here: if the attribute isn't at the path; then that's arguably a good thing right? We don't need to delete

Reply:
yes, good point, changing....